README.md

```diff
@@ -287,3 +288,78 @@ import { isLoaded } from 'laravel-vue-i18n';
 const loaded = isLoaded(); // true
 const loaded = isLoaded('fr'); // false
 ```
+
+## Using multiple instances
+
+Under the hood, the Vue plugin is using a `I18n` class which encapsulates all the translation logic and the currently active language. This means that it's possible to create multiple class instances, each with different options and active language. This can be useful for scenarios where part of the app needs to be translated to a language different from the main UI.
+
+Note that loaded languages are still shared between different instances. This avoids loading the same set of translations multiple times. The main difference between different instances will be the currently active language.
```

Comment:
@xiCO2k one thing I realized as I was writing this: it may actually be desirable to _not_ shared loaded languages between different instances. This would allow splitting language files into smaller chunks - perhaps by feature - and then load only the relevant language file for a particular part/region of the app.

However, I personally don't think it would be a blocker for merging what we already have. Perhaps an idea for later, though?

Reply:
Sure thing.